docs/advanced_schemas.md

```diff
@@ -8,6 +8,6 @@ statics, and virtuals from an ES6 class. A class method maps to a schema
 method, a static method maps to a schema static, and getters/setters map
 to virtuals.
 
-```javascript
+```acquit
```

Comment:
Does syntax highlighting still work with this change?

Reply:
yes, in `website.js` the codeblock (with acquit) get replaced with javascript ones before markdown processing

this was necessary because eslint would complain about invalid tokens